plugins/by-name/nerdy/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, config, ... }:
+lib.nixvim.plugins.mkNeovimPlugin {
+  name = "nerdy";
+  packPathName = "nerdy.nvim";
+  package = "nerdy-nvim";
+
+  maintainers = [ lib.maintainers.GaetanLepage ];
+
+  callSetup = false;
+  hasSettings = false;
+
+  extraOptions = {
+    enableTelescope = lib.mkEnableOption "telescope integration";
+  };
+
+  extraConfig = cfg: {
+    assertions = lib.nixvim.mkAssertions "plugins.git-worktree" {
+      assertion = cfg.enableTelescope -> config.plugins.telescope.enable;
+      message = ''
+        You have to enable `plugins.telescope` as `enableTelescope` is activated.
```

Comment:
Again: phrasing seems off

Reply:
Normally this is phrased as https://github.com/nix-community/nixvim/blob/db01a0dd9703e002058ffd805bbe48c4c30742a0/plugins/by-name/refactoring/default.nix#L32-L35

E.g:

> You have enabled the `telescope` integration with refactoring-nvim.
> However, you have not enabled the `telescope` plugin itself (`plugins.telescope.enable = true`).

Reply:
Actually, this is very inconsistent treewide. So nevermind.